var/spack/repos/builtin/packages/py-atlalign/package.py

```diff
@@ -32,3 +32,6 @@ class PyAtlalign(PythonPackage):
     depends_on('py-tensorflow-addons@0.12.1', type=('run'))
 
     patch('lpips.patch', when='@0.6.0')
+
+    def patch(self):
+        filter_file('"tensorflow>=[0-9.]+",', '', 'setup.py')
```

Comment:
I'm not sure if this is enough to trigger a rebuild of the package.

Reply:
Thanks for raising this point. So I'll use the same trick as the one of `py-tensorflow-addons/package.py`:  add a new version with the same `sha256`. See next commit.